style.css

```diff
@@ -400,3 +429,129 @@ textarea::placeholder {
   width: 131px;
   margin: 2px auto;
 }
+
+.footer-icons {
+  justify-content: center;
+  min-height: 71px;
+  border-top: 1px solid #dfe1e6;
+  color: #505f79;
+}
+
+.desc {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.about-flex-container {
+  margin-bottom: 100px;
+}
+
+@media (min-width: 768px) {
+  .menu-bar-desktop {
+    display: flex;
+  }
+
+  .menu-icon {
+    display: none;
+  }
+
+  .headline {
```

Comment:
The background image of the headline is overlapping the text on the small mobile view. Kindly make sure that the mobile version works the same as expected and according to the design.

![back](https://user-images.githubusercontent.com/35504149/171647988-9182a805-6e69-42f3-a95f-0a3fcec34d7b.png)